src/main/groovy/com/blackducksoftware/integration/hub/dataservice/report/RiskReportDataService.java

```diff
@@ -90,6 +92,43 @@ public RiskReportDataService(final IntLogger logger, final RestConnection restCo
 
     }
 
+    public String getNoticesReportData(final String projectName, final String projectVersionName) throws IntegrationException {
+        final ProjectView project = projectRequestService.getProjectByName(projectName);
+        final ProjectVersionView version = projectVersionRequestService.getProjectVersion(project, projectVersionName);
+        return getNoticesReportData(project, version);
+    }
+
+    public String getNoticesReportData(final ProjectView project, final ProjectVersionView version) throws IntegrationException {
+        logger.trace("Getting the Notices Report Contents using the Report Rest Server");
+        return reportRequestService.generateHubNoticesReport(version, ReportFormatEnum.TEXT);
+    }
+
+    public File createNoticesReportFile(final File outputDirectory, final String projectName, final String projectVersionName) throws IntegrationException {
+        return createNoticesReportFile(outputDirectory, getNoticesReportData(projectName, projectVersionName), projectName, projectVersionName);
+    }
+
+    public File createNoticesReportFile(final File outputDirectory, final ProjectView project, final ProjectVersionView version) throws IntegrationException {
+        return createNoticesReportFile(outputDirectory, getNoticesReportData(project, version), project.name, version.versionName);
+    }
+
+    private File createNoticesReportFile(final File outputDirectory, final String noticesReportContent, final String projectName, final String projectVersionName) throws HubIntegrationException {
+        final IntegrationEscapeUtil escapeUtil = new IntegrationEscapeUtil();
```

Comment:
Just for fun, we should probably create this in HubServicesFactory and pass it in to data service constructors where needed.